classes/controllers/FrmInboxController.php

```diff
@@ -49,16 +48,30 @@ public static function dismiss_all_button( $atts ) {
 	 * @return void
 	 */
 	public static function inbox() {
-		FrmAppHelper::include_svg();
+		_deprecated_function( __METHOD__, '6.7.1' );
+	}
+
+	/**
+	 * @since x.x
+	 *
+	 * @return array
+	 */
+	public static function get_inbox_messages() {
 		self::add_tracking_request();
 		self::add_free_template_message();
 
-		$inbox    = new FrmInbox();
-		$messages = $inbox->get_messages( 'filter' );
-		$messages = array_reverse( $messages );
-		$user     = wp_get_current_user();
+		$inbox              = new FrmInbox();
+		$unread_messages    = $inbox->get_messages();
+		$dismissed_messages = $inbox->get_messages();
```

Comment:
Let's reuse the messages already fetched instead of calling the same code again.
```suggestion
		$dismissed_messages = $unread_messages;
```